src/unity/python/doc/source/turicreate.toolkits.style_transfer.rst

```diff
@@ -0,0 +1,16 @@
+:mod:`style_transfer`
+==========================
+
+.. automodule:: turicreate.toolkits.style_transfer
+
+.. currentmodule:: turicreate.style_transfer
+
+
+object detector
```

Comment:
This line needs to be change, probably to `style transfer`.

Reply:
Nice catch, fixed it